src/Uno.Core/Options/Option.cs

```diff
@@ -29,7 +29,7 @@ public abstract class Option
 		/// <summary>
 		/// Creates an option for a given value.
 		/// </summary>
-		public static Some<T> Some<T>(T value) => new Some<T>(value);
+		public static Option<T> Some<T>(T value) => new Some<T>(value);
```

Comment:
This is a binary breaking change.

Reply:
Yep it is. But it should be completed anyway.